### ISSUE TITLE
Some missing units

### ISSUE THREAD

**Issue by FEA-eng** [Proektsoftbg/Calcpad#252]
Hi,

I've noticed that the following units are not available in Calcpad and would be nice to have:
- dB
- Gs (Gauss)
- b, kb, Mb, Gb, Tb, Pb, B, kB, MB, GB, TB, PB
- kmol, nmol and maybe other multiples
- Mx (Maxwell)
- week, month, year
- pc (Parsec)
- Å (Angstrem)

**Comment by Proektsoftbg** [comment]
Hi! Thank you for your suggestions.

Some of them we already have. Week is "w" and year is "y". I doubted about month before because different months have different number of days - 28(29), 30 and 31. Probably 30 days would be OK, but there is some ambiguity.

And what is "b" an "B"?

**Comment by FEA-eng** [comment]
> Some of them we already have. Week is "w" and year is "y". 

Right, they aren't listed on the main page (https://github.com/Proektsoftbg/Calcpad) so I thought they aren't available.

> I doubted about month before because different months have different number of days - 28(29), 30 and 31. Probably 30 days would be OK, but there is some ambiguity.

I've seen the value of 30.4368499 days used in some cases. Like in SMath or Google if you search for "month to days".

> And what is "b" an "B"?

b - bit, B - byte

**Comment by Proektsoftbg** [comment]
OK. Thank you. Probably I have forgotten to add "w" and "y" to this readme at the time they were implemented. I will update it accordingly,

About SMath, probably they defined month as average = year/12.

<img width="129" alt="image" src="https://github.com/Proektsoftbg/Calcpad/assets/90918363/fdcc0e57-e571-428a-aeb5-abcebf10f6e1">

In this respect, probably we have to reconsider the unit for years  "y" in Calcpad. Now it is 365 days.
In SMath, it is 365.2425 days, which is according to the Gregorian calendar:
https://en.wikipedia.org/wiki/Leap_year

However, in MathCAD, it is 365.24219878125 days, which is probably something like an exact "tropical" year duration:
https://en.wikipedia.org/wiki/Tropical_year

<img width="245" alt="image" src="https://github.com/Proektsoftbg/Calcpad/assets/90918363/6c443355-d643-43a7-8e21-107449013ab0">

There is also "astronomical" year, which is defined as 365.25 days by convention:
https://multiverse.ssl.berkeley.edu/Portals/0/CalendarInTheSky/Resources/Presentations/HowLongIsAYear.pdf?ver=Wt-kX9xEaM0fZve8OXxBfg%3d%3d

To be honest, as a structural engineer I am getting confused by having so many "years". The question is what is the most appropriate to assume in Calcpad. The SMath approach seems to be more practical and simpler for everyday engineering calcs.

**Comment by FEA-eng** [comment]
Even more definitions here: https://www.physicsforums.com/insights/measuring-how-many-days-are-in-a-year/

I think that I would choose the approach used in SMath too.

**Comment by Proektsoftbg** [comment]
OK. Thank you! I will make the required changes.
